src/foam/nanos/medusa/sf/MedusaSFManager.js

```diff
@@ -44,6 +62,24 @@ foam.CLASS({
       name: 'medusaFileCapacity',
       value: 4096
     },
+    {
+      class: 'String',
+      name: 'folderName',
+      value: '../saf/'
+    },
+    {
+      class: 'Int',
+      documentation: 'set -1 replay nothing; set to MAXIMUM_REPLAY_DAYS replay all',
+      name: 'replayStrategy',
+      units: 'days',
+      javaSetter: `
+        replayStrategyIsSet_ = true;
+        if ( val < 0 ) replayStrategy_ = -1;
+        else if ( val >=  getMAXIMUM_REPLAY_DAYS() ) replayStrategy_ = getMAXIMUM_REPLAY_DAYS();
```

Comment:
```suggestion
        else if ( val >=  MAXIMUM_REPLAY_DAYS ) replayStrategy_ = getMAXIMUM_REPLAY_DAYS();
```

Reply:
MAXIMUM_REPLAY_DAYS is PropertyInfo in java